docs/operator/policy_packages.md

```diff
@@ -95,6 +98,28 @@ In all cases the names used to register the functions must be prefixed
 with the name of the virtual organisation that owns the policy package,
 to avoid naming conflicts on multi-VO Rucio installations.
 
+### lfn2pfn vs. non_deterministic_pfn algorithms
+
+`lfn2pfn` algorithms and `non_deterministic_pfn` algorithms are
```

Comment:
I think we need to specify this a bit more. People might ask, why is one predictable and one is not, after all a computer can generate both of them.
The difference is that the lfn2pfn can just be "constructed" by knowing `scope` and `name` of a file (Plus protocol specific information of the RSE). While the `non_deterministic_pfn` can use additional information, such as the dataset the file is in, or metadata, etc.

Reply:
Good point. I have tried to make this distinction clearer.